Makefile

```diff
@@ -409,9 +418,9 @@ tools/$(HTML_COMPRESSOR):
 	cd tools; wget --no-check-certificate https://htmlcompressor.googlecode.com/files/$(HTML_COMPRESSOR) -O $(HTML_COMPRESSOR)
 else
 tools/$(HTML_COMPRESSOR):
-	$(Q) mkdir -p tools
-	cd tools; wget https://github.com/yui/yuicompressor/releases/download/v2.4.8/$(YUI_COMPRESSOR)
-	cd tools; wget https://htmlcompressor.googlecode.com/files/$(HTML_COMPRESSOR)
+#	$(Q) mkdir -p tools
+#	cd tools; wget https://github.com/yui/yuicompressor/releases/download/v2.4.8/$(YUI_COMPRESSOR)
+#	cd tools; wget https://htmlcompressor.googlecode.com/files/$(HTML_COMPRESSOR)
```

Comment:
Exact error message is
```
cd tools; wget https://htmlcompressor.googlecode.com/files/htmlcompressor-1.5.3.jar
--2016-09-16 21:27:30--  https://htmlcompressor.googlecode.com/files/htmlcompressor-1.5.3.jar
Resolving htmlcompressor.googlecode.com (htmlcompressor.googlecode.com)... 2a00:1450:400c:c09::52, 64.233.166.82
Connecting to htmlcompressor.googlecode.com (htmlcompressor.googlecode.com)|2a00:1450:400c:c09::52|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2016-09-16 21:27:30 ERROR 404: Not Found.

Makefile:417: recipe for target 'tools/htmlcompressor-1.5.3.jar' failed
make: *** [tools/htmlcompressor-1.5.3.jar] Error 8
```
And it is not going away. Path to htmlcompressor is not working anymore!